documentation/staging/content/userguide/managing-operators/using-helm.md

```diff
@@ -96,6 +96,18 @@ Example:
 javaLoggingLevel:  "FINE"
 ```
 
+##### `kubernetesPlatform`
+Allows you to set the Kubernetes platform on which the operator is running. The value is case-insensitive.
```

Comment:
Please consider replacing this with something like the following slight rewrite of the current wording in order to change its emphasis and include a little additional info: 

Specify the Kubernetes platform on which the operator is running. This setting has no default, the only valid value is `OpenShift`, and the setting should be left unset for other platforms. When set to `OpenShift`, the operator:

- Sets the domain home file permissions in each WebLogic Server pod to work correctly in OpenShift for [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), and [Domain home in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) domains. Specifically, it sets file group permissions so that they match file user permissions. 

- Sets the `weblogic.SecureMode.WarnOnInsecureFileSystem` Java system property to `false` on the command line of each WebLogic Server. This flag suppresses insecure file system warnings that are reported in the WebLogic Server console when the WebLogic Server is in production mode. These warnings result from setting the file permissions that are necessary to work with restricted security context constraints on OpenShift.

For more information about the security requirements for running WebLogic in OpenShift, see the [OpenShift chapter]({{<relref "/security/openshift.md">}}) in the Security section.

Example:
```yaml
kubernetesPlatform: Openshift
```